hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/transfer/AutoAccountCreator.java

```diff
@@ -121,8 +122,12 @@ public AccountID create(@NonNull final Bytes alias, final boolean isByTokenTrans
         //                .build();
         //        accountStore.put(payerCopy.copyBuilder().build());
 
+        // TODO: Check if this is the correct verifier
+        final Predicate<Key> verifier =
+                key -> handleContext.verificationFor(key).passed();
+
```

Comment:
Might be helpful to have a proper class (e.g. `DefaultSignatureVerifier`) that implements this "default" verifier Predicate so we aren't creating unnecessary lambdas in the common case where we just want this default behavior.

Reply:
Good idea! Will do.